Mathlib/MeasureTheory/Measure/Portmanteau.lean

```diff
@@ -77,13 +78,24 @@ probability measure
 
 noncomputable section
 
-open MeasureTheory
+open MeasureTheory Set Filter BoundedContinuousFunction
 
-open Set
+-- Q: Where do these belong? (Tried `Topology.Instances.Real` and `Order.LiminfLimsup` and
+-- `Topology.Algebra.Order.LiminfLimsup` but none worked. `#find_home` suggests the present file.)
```

Comment:
I haven't found a good file. You could create a new one, in `Topology.Order`. For sure, it shouldn't be in the current file!